functions/firebase/firestore-reactive/src/test/java/functions/FirebaseFirestoreReactiveTest.java

```diff
@@ -92,6 +92,25 @@ public void functionsFirebaseReactive_shouldCapitalizeOriginalValue()  {
         "Replacing value: foo --> FOO");
   }
 
+  @Test
+  public void functionsFirebaseReactive_shouldIgnoreCapitalizedValues()  {
+
+    String jsonStr = gson.toJson(Map.of("value",
+        Map.of("fields",
+            Map.of("original",
+                Map.of("stringValue", "FOO")))));
+
+    MockContext context = new MockContext();
+    context.resource = "projects/_/databases/(default)/documents/messages/ABCDE12345";
```

Comment:
It would be good to explain this a bit.  I know it's in a test and not published, but it's not obvious.